pkg/fanal/analyzer/language/nodejs/yarn/yarn.go

```diff
@@ -25,7 +27,7 @@ import (
 	"github.com/aquasecurity/trivy/pkg/fanal/analyzer"
 	"github.com/aquasecurity/trivy/pkg/fanal/analyzer/language"
 	"github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/nodejs/license"
-	"github.com/aquasecurity/trivy/pkg/fanal/types"
+	ftypes "github.com/aquasecurity/trivy/pkg/fanal/types"
```

Comment:
Any reasons for renaming it?

Reply:
We often use `types` and `ftypes` in the same file, so I'm used to using `ftypes`.
To make it more uniform - I made this change.

Reply:
Ideally, we should not use `pkg/types` under `fanal`. So, they will not conflict. Also, I think we can rename it once they actually conflict.

Reply:
If you want to rename it, I don't mind it, but can we change it in another PR? I want to focus on changes for this enhancement. It currently looks like there are many changes on GitHub, and I need to carefully distinguish which lines are changes due to `ftypes` and which lines are related to the actual feature additions.  

Additionally, for a personal reason, since I am reviewing this while traveling and do not have a large display, I have to scroll a lot to skip the `ftypes` changes.

Reply:
No problem. It's not really important for me.
Also you're right - yarn analyzer package is already in `fanal,` so it's not necessary to point it out again.

I returned `types` in https://github.com/aquasecurity/trivy/pull/8535/commits/9fb33ff6c426e786dd49036a6546e5a42d5769fb